scripts/test

```diff
@@ -1,20 +1,29 @@
-#!/bin/sh
+#!/bin/bash
 
-set -e
+declare -A test_results
+
+DOCKER_TLS_CERTDIR=""
+dockerd -H unix:///tmp/install-docker.sock & &>/dev/null
+
+export DOCKER_HOST="unix:///tmp/install-docker.sock"
+
+while true; do
+  docker version && break
+  sleep 5
+done
 
 echo "==> Testing install-docker scripts..."
 
-# https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string
-SEMVER_REGEX='^(\d*)\.(\d*)\.(\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$'
```

Comment:
Why `SEMVER_REGEX` is removed?? It was here to assure that just `pkg/<SEMVER>` are processed.

Reply:
What is the situation where a non semver ends up in `pkg/` and this check is needed?

Reply:
I think this check is needed as we are releasing SEMVER notation named scripts, `pkg` are building `dist` files. I don't think is a bad practice to check that naming is using SEMVER notation. What's the reason to remove it?? 

Anyway, if `SEMVER_REGEX ` filter is removed, the `README.md` should also be updated as https://github.com/rancher/install-docker#add-new-docker-release is saying `...version should follow semver format...`